src/components/dashboard.tsx

```diff
@@ -1,5 +1,3 @@
-import React from 'react';
-
 function Dashboard() {
```

Comment:
**praise**: The functional component is simple and clean, making it easy to understand and maintain.